lib/tasks/reports.rake

```diff
@@ -14,8 +14,8 @@ namespace :reports do
     puts "Running discovery report analytics for #{total} discovery report job runs"
     CSV.open(output_file, 'w') do |csv|
       csv << %w[num_objects num_files num_errors runtime_minutes user report_date]
-      JobRun.where(job_type: 'discovery_report').each.with_index do |job_run, index|
-        puts "#{index + 1} of #{total}"
+      JobRun.where(job_type: 'discovery_report').each.with_index(1) do |job_run, i|
+        puts "#{i} of #{total}"
         if job_run.output_location && File.exist?(job_run.output_location)
           num_found += 1
```

Comment:
is this line a correction for 0-based?  should it be kept or not?

Reply:
no change in behavior, it was just changed to be consistent with the code above (passing `(1)` to `with_index` instead of adding 1 to the index in the loop .... it also changes the variable name to `i` for consistency with other parts of the code

Reply:
sorry I mean line 20, and I now see that my concern/comment was silly.